README.md

```diff
@@ -92,11 +92,13 @@ the name prefix. List of variables:
 - `network_connections` - The connection profiles are configured as
   `network_connections`, which is a list of dictionaries that include specific
   options.
-- `network_allow_restart` - Certain configurations require the role to restart
-  network services. For example, if a wireless connection is configured and
-  NetworkManager-wifi is not installed, NetworkManager must be restarted prior
-  to the connection being configured. Setting this to `false` will prevent the
-  role from restarting network service.
+- `network_allow_restart` - It defaults to `false`. Certain configurations like
+  wireless or team connections require the role to restart network services.
+  For example, if a wireless connection is configured and NetworkManager-wifi
+  is not installed, NetworkManager must be restarted prior to the connection
+  being configured, and the user needs to explicitly consent to it by setting
```

Comment:
Please mention that the restart can result in connectivity loss and therefore the role does not do this without the consent.